Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -568,16 +534,23 @@ CGPathRef CGPathRetain(CGPathRef path) {
 }
 
 /**
-@Status Stub
-@Notes
+@Status Interoperable
 */
 void CGPathAddQuadCurveToPoint(CGMutablePathRef path, const CGAffineTransform* transform, CGFloat cpx, CGFloat cpy, CGFloat x, CGFloat y) {
-    UNIMPLEMENTED();
+    RETURN_IF(!path);
+
+    CGPoint endPoint = __CreateCGPointWithTransform(x, y, transform);
```

Comment:
>__CreateCGPointWithTransform [](start = 23, length = 28)

I am worried about transformed points without transforming the beziers somehow as well.. hmm.